model.mod

```diff
@@ -1,11 +1,48 @@
 # Default data
 param number_of_laps;
 set race_length := 1..number_of_laps;
-set lap_options;
+set race_length2 := 1..number_of_laps-1;
```

Comment:
ezt nem vezetnem be:
1) semmit sem mond ez a neve
2) osszesen ket helyen hasznalod, ertelmesebb lenne, ha ott lenne `lap in race_length2` helyett `lap in race_length : lap != number_of_laps`